hexrd/material.py

```diff
@@ -525,6 +525,9 @@ def _set_sgnum(self, v):
         # Update the unit cell if there is one
         if hasattr(self, 'unitcell'):
             self.unitcell.sgnum = v
+            self.unitcell.getHKLs(self.dmin.getVal('nm'))
+            self._newPdata()
```

Comment:
It looks like `self._newPdata()` calls `self.unitcell.getHKLs(self._dmin.getVal('nm'))` as well.

Do we need the `getHKLs()` here?

Reply:
Oh right, yes good catch. We can remove that.